frontend/packages/data-portal/app/components/Viewer/ViewerPage.tsx

```diff
@@ -344,18 +385,62 @@ export function ViewerPage({
           />
         </div>
         {/* Add empty space to push content to right */}
-        <div className="basis-sds-xxl flex-grow md:mr-sds-xxl" />
+        <div className="basis-sds-xxl flex-grow lg:mr-sds-default xl:mr-sds-xxl" />
         <div className="flex basis-auto flex-shrink-0">
           <div className="flex items-center pt-1 gap-[1px] sm:gap-1 sm:pt-0">
+            {shouldShowTomogramDropdown && (
+              <NeuroglancerDropdown title="Tomograms" variant="outlined">
+                <MenuDropdownSection title="Select tomogram">
+                  {tomograms.map((tomogram) => {
+                    return (
+                      <NeuroglancerDropdownOption
+                        key={tomogram.id.toString()}
+                        selected={selectedTomogram(tomogram)}
+                        disabled={unsupportedTomogramSwitch(tomogram)}
+                        onSelect={() => {
+                          if (selectedTomogram(tomogram)) return
+                          voxelSpacing.current = tomogram.voxelSpacing
```

Comment:
This could be extracted into another function, probably a little complex inline here